### PR TITLE
Added UART

### PR DESCRIPTION
**Pull request by rs2thoma**

### CODE REVIEW

**Comment by amc1234** [comment]
Did you test this with a UART device?

File: uart.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import serial
+
+UART_0 = '/dev/ttyS0'
+UART_1 = '/dev/ttyAMA0'
+
+class UART():
+    def __init__(self, port, baud):
+        self.ser = serial.Serial(port, baud)
```
Comment (tiger-hao) [inline uart.py]: can we give this a better name than `ser`? any reason we can't just call it `serial`?

File: uart.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import serial
```
Comment (tiger-hao) [inline uart.py]:
```suggestion
from serial import Serial
```

**Review by tiger-hao — APPROVED** [review]
code looks good

**Review by amc1234 — APPROVED** [review]

File: uart.py (hunk 1 above)
Comment (rs2thoma) [reply uart.py]: its an extra 3 chars to type :)

**Pull request merged**